src/components/form.tsx

```diff
@@ -2,7 +2,9 @@ import * as React from 'react';
 import { Link } from 'react-router-dom';
 
 import { default as Button, ButtonStyle } from './button';
-import InputField from './input_field'
+
+
+export type FormValidationResponse = { [key: string]: any[] | string }
```

Comment:
maybe it's nice to explain why it can be any[] cause I also asked myself the same thing today :stuck_out_tongue_closed_eyes: 

We also might just use `any`